nss-passwords.yaml

```diff
@@ -0,0 +1,109 @@
+package:
+  name: nss-passwords
+  version: 0.6
+  epoch: 0
+  description: "Read Mozilla keyrings from command line"
+  copyright:
+    - license: MPL-1.1
+    - license: GPL-2.0-only
+    - license: LGPL-2.1-or-later
+  dependencies:
+    runtime:
+      - libnss
+
+environment:
+  contents:
+    packages:
+      - apk-tools
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - git
+      - libnss-dev
+      - mercurial
+      - ocaml
+      - ocaml-compiler-libs
+      - opam
+      - openssl-dev
+      - patch
+      - posix-libc-utils
+      - rsync
+      - sqlite-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/glondu/nss-passwords
+      tag: ${{package.version}}
+      expected-commit: c2dd491aae707bd4514d936ea52e13ed6eda141c
+
+  - runs: |
+      opam init --disable-sandboxing --shell=sh --disable-shell-hook -y
+      eval $(opam env)
+      opam switch create build-switch ocaml-base-compiler.5.3.0 -y
```

Comment:
Can we get this version dynamically? Or at least define it in a variable above with an explanation of where it comes from?